forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/ApplicationPinFragment.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+package org.forgerock.android.auth.devicebind
+
+import android.content.DialogInterface
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import androidx.fragment.app.DialogFragment
+import org.forgerock.android.auth.databinding.FragmentApplicationPinBinding
+
+/*
+Default Dialog fragment to request user for Application Pin
+ */
+class ApplicationPinFragment : DialogFragment() {
```

Comment:
this class looks good..

Reply:
i think we can apply the same binding code part for DeviceBindFragment. i like this class better. should be a simple fix

    private var _binding: FragmentUserDeviceBindBinding? = null
    private val binding get() = _binding!!

Reply:
Updated with view binding.